stdlib/2and3/optparse.pyi

```diff
@@ -136,7 +136,7 @@ class OptionContainer:
     def _create_option_mappings(self) -> None: ...
     def _share_option_mappings(self, parser: OptionParser) -> None: ...
     def add_option(self, *args, **kwargs) -> Any: ...
-    def add_options(self, option_list: List[Option]) -> None: ...
+    def add_options(self, option_list: Sequence[Option]) -> None: ...
```

Comment:
This is just iterated over in a for loop. So `Iterable[Option]` may be more correct.